controllers/api/post-routes.js

```diff
@@ -0,0 +1,128 @@
+const router = require("express").Router();
+
+const { User, Post, Comment } = require("../../models");
+
+// Get all posts
+router.get("/", async (req, res) => {
+  try {
+    const allPosts = await Post.findAll({
+      attributes: ["id", "title", "content", "created_at"],
+      order: [["created_at", "DESC"]],
```

Comment:
I think the order has to be ASC here, or maybe it didn't apply properly as the posts at the top are the first posts that were created.
Nice thinking though :-)